src/plugins/saved_objects_tagging_oss/public/api.ts

```diff
@@ -268,6 +268,11 @@ export type SavedObjectSaveModalTagSelectorComponentProps = EuiComboBoxProps<
    * tags selection callback
    */
   onTagsSelected: (ids: string[]) => void;
+
+  /**
+   * Add "Optional" to the label
+   */
+  markOptional?: boolean;
```

Comment:
Can you explain what `markOptional` means ? What is optional? The tag? It seems that it is a prop of `SavedObjectSaveModalTagSelectorComponentProps` but I am not sure what it means in this context.

Reply:
Hi @sebelga it adds this "Optional" text

<img width="558" alt="Screenshot 2023-05-30 at 1 05 51 PM" src="https://github.com/elastic/kibana/assets/315764/2b18787e-0257-4368-8120-7d43629aa3be">

Reply:
Ok thanks for clarifying... Naming is hard... but not finding better alternative. 😊